software/GPSDO.ino

```diff
@@ -396,22 +396,25 @@ void cmd_tunnel(SerialCommands* sender)
 
 // called for SP (set PWM) command
 void cmd_setPWM(SerialCommands* sender)
-{
-  uint16_t pwm;
+{ 
+  uint32_t pwm;
```

Comment:
2^16 = 65536
Why 32?

Reply:
Hi Felixd,
I had to use uint32_t because otherwise by entering a number greater than 65535 (by error from keyboard) it was trimmed by entering a totally incorrect PWM value.
They are ideas to evaluate but in my opinion useful to avoid errors.

Reply:
Because it is not a number but a string, initially.
Then the atoi function converts it into integer.

Reply:
Thank you for your contribution, iannezsp! I have manually merged this in Version v0.05e. Note there was a small mistake in the patch as submitted, which I have corrected.